configure.ac

```diff
@@ -114,6 +114,12 @@ AC_ARG_WITH(sim,
 	[with_sim=qemu]
 	)
 
+AC_ARG_WITH(languages,
+        [AS_HELP_STRING([--with-languages=],
+                [Sets the enabled languages for GNU toolchain, defaults to c,c++,fortran for Linux/native and Linux/glibc or c,c++ for Bare-metal, Linux/musl and Linux/uClibc])],
```

Comment:
GCC rather tha GNU?

Reply:
I think the term GNU toolchain is clear. If it turns out to be too confusing, we might adjust.